trieste/models/gpflow/models.py

```diff
@@ -83,10 +85,13 @@ def __init__(
         :param num_kernel_samples: Number of randomly sampled kernels (for each kernel parameter) to
             evaluate before beginning model optimization. Therefore, for a kernel with `p`
             (vector-valued) parameters, we evaluate `p * num_kernel_samples` kernels.
-        :param num_rff_features: The number of random Foruier features used to approximate the
+        :param num_rff_features: The number of random Fourier features used to approximate the
             kernel when calling :meth:`trajectory_sampler`. We use a default of 1000 as it
             typically perfoms well for a wide range of kernels. Note that very smooth
             kernels (e.g. RBF) can be well-approximated with fewer features.
+        :param used_decoupled_sampler: If True use a decoupled random Fourier feature sampler, else
+            just use a random Fourier feature sampler. The decoupled sampler is suffers less from
+            overestimating variance and can typically get away with a lower num_rff_features.
```

Comment:
(is there any reason to ever use a non-decoupled sampler then?)

Reply:
Perhaps sometimes a  little faster and a useful baseline